bossbot/db.py

```diff
@@ -10,6 +10,7 @@
 LASTFM_USER_TABLE = 'CREATE TABLE LASTFM_USERS(user_id INTEGER PRIMARY KEY, lastfm_username TEXT)'
 KARMA_TRACKER_TABLE = 'CREATE TABLE KARMA_TRACKER(user_id INTEGER PRIMARY KEY, karma INTEGER)'
```

Comment:
since we're just tracking positive karma and there's a possibility to add tracking individual increases/decreases to karma later, can we  change this from `karma INTEGER` to something like `positive_karma INTEGER`?

Reply:
impossible: https://www.youtube.com/watch?v=SlPwX1VJ7H8

also yeah, plus_karma and minus_karma SGTM